io/subsystem/src/main/java/org/wildfly/extension/io/logging/IOLogger.java

```diff
@@ -75,4 +75,7 @@
 
     @Message(id = 7, value = "Unexpected bind address conflict in resource \"%s\" when attempting to establish binding for destination %s to %s: a binding of %s already existed")
     OperationFailedException unexpectedBindAddressConflict(PathAddress currentAddress, CidrAddress cidrAddress, InetSocketAddress bindAddress, InetSocketAddress existing);
+
+    @Message(id = 8, value = "Please proivde a realistic stack-size value")
```

Comment:
s/proivde/provide

Also, please pass in the value they provided and include it in the message. "The stack-size value of %d bytes for IO worker %s is low and may result in problems. A value of at least 150,000 is recommended."

Also this needs the @LogMessage annotation and the method return type should be 'void'. That's how methods that produce log messages are handled. Methods without that are not for log messages; they return objects that include localized strings and then the code that calls the method does something with the object.

Reply:
Thank you for the review :) I have changed it accordingly.